app/controllers/teams_controller.rb

```diff
@@ -0,0 +1,112 @@
+class TeamsController < ApplicationController
+  # Preload organization
+  before_action :organization
+
+  def new
+    @team = Team.new
+  end
+
+  def edit
+    team
+    fetch_possible_new_users
+    @new_projects = organization.projects.where.not(id: team.projects)
+  end
+
+  def create
+    @team = Team.new(team_params.merge(organization_id: organization.id))
+    if team.save
+      redirect_to_edit_team(:success, t('teams.create.success'))
+    else
+      render 'new'
+    end
+  end
+
+  def update
+    if team.update(team_params)
+      redirect_to_edit_team(:success, t('teams.update.success'))
+    else
+      render 'new'
+    end
+  end
+
+  def destroy
+    if @team == @organization.admin_team
```

Comment:
Its wrong, shoud be `team`. This validation is only defensive, you can't do it from the app